setuptools/tests/test_sdist.py

```diff
@@ -822,6 +823,20 @@ def get_source_files(self):
         manifest = cmd.filelist.files
         assert '.myfile~' in manifest
 
+    def test_build_base_pathlib(self, source_dir):
```

Comment:
Is it acceptable to skip this test on the deprecated `distutils` from stdlib?

Reply:
Oh, right. Hmm. There's a good chance the concerns don't intersect, so let's skip the test for now and revive it if a user reports issues with that arrangement.